includes/class-wc-stripe-helper.php

```diff
@@ -255,6 +260,22 @@ public static function no_decimal_currencies() {
 		];
 	}
 
+	/**
+	 * List of currencies supported by Stripe that has three decimals
+	 * https://docs.stripe.com/currencies?presentment-currency=AE#three-decimal
+	 *
+	 * @return array $currencies
+	 */
+	private static function tree_decimal_currencies() {
```

Comment:
```suggestion
	private static function three_decimal_currencies() {
```

Small typo in the function name :)